src/components/templates/ReactionPage/ReactionList.tsx

```diff
@@ -0,0 +1,107 @@
+import React, { useMemo, useState } from "react";
+import classNames from "classnames";
+
+import { DEFAULT_PARTY_NAME, DEFAULT_PROFILE_IMAGE } from "settings";
+
+import {
+  chatMessageAsMessageToTheBand,
+  Reaction,
+  ReactionsTextMap,
+} from "utils/reactions";
+import { withId, WithId } from "utils/id";
+
+import { User } from "types/User";
+import { ChatMessage } from "types/chat";
+
+import { useWorldUsersByIdWorkaround } from "hooks/users";
+
+import UserProfileModal from "components/organisms/UserProfileModal";
+import UserProfilePicture from "components/molecules/UserProfilePicture";
+import { UserAvatar } from "components/atoms/UserAvatar";
+
+export interface ReactionListProps {
+  reactions: Reaction[];
+  chats: ChatMessage[];
```

Comment:
should this be chatMessages/similar?

Reply:
Could be, happy to change it. This is just how it was defined in the existing component.

Reply:
fixed in https://github.com/sparkletown/sparkle/pull/1215/commits/46b090f79f949a09c4a68254188b947da2b058a6